### PR TITLE
add desctop and mobile header

### PR DESCRIPTION
**Pull request by oksana-liseczkaya**

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -8,12 +8,72 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" />
```
Comment (ghost) [inline Triphouse-project/index.html:8]: 2021-04-19_22h20_28
Move image to Triphouse-project folder

Comment (ghost) [reply Triphouse-project/index.html:8]: After fix path to image in html and css.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -8,12 +8,72 @@
...
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="./style.css" />
-    <title>Document</title>
+    <title>Triphouse</title>
 </head>
-
 <body>
     <header>
```
Comment (ghost) [inline Triphouse-project/index.html:14]: 2021-04-19_22h22_52
The background image should cover the whole page width.

File: Triphouse-project/index.html [hunk 3]
```diff
@@ -8,12 +8,72 @@
...
+        <div class="header__main">
+            <div class="header__menu">
```
Comment (ghost) [inline Triphouse-project/index.html:17]: 2021-04-19_22h24_57
https://t.me/c/1421457090/1292
Investigate example, your variant of implementation isn't good.

File: Triphouse-project/index.html (hunk 1 above)
Comment (oksana-liseczkaya) [reply Triphouse-project/index.html:8]: Moved img to Triphouse-project folder and fixed path

File: Triphouse-project/index.html (hunk 2 above)
Comment (oksana-liseczkaya) [reply Triphouse-project/index.html:14]: Fixed background image

File: Triphouse-project/index.html (hunk 3 above)
Comment (oksana-liseczkaya) [reply Triphouse-project/index.html:17]: Fixed underline

**Pull request merged**